scripts/modules/service.py

```diff
@@ -57,6 +58,10 @@ def __init__(self, **options):
                 self.apm_api_key = {"ELASTIC_APM_API_KEY": options.get("elastic_apm_api_key")}
             else:
                 print('WARNING: elastic_apm_api_key is not supported for the current version. Use version +7.6.')
+        if self._oss and self.name() in ("elasticsearch", "kibana"):
+            if self.at_least_version("7.11") or (self.at_least_version("6.8.14") and self.version_lower_than("6.9")):
+                print('ERROR: OSS distribution is ONLY supported in 7.11+/6.8.14+ for Kibana and Elasticsearch.')
+                sys.exit(1)
```

Comment:
💯 Thanks for acknowledging my paranoia. ;) This makes me feel much better.

Reply:
I was not 100% confident about my first attempt, thanks for challenging it!!